fastlane/lib/fastlane/plugins/template/.rubocop.yml

```diff
@@ -0,0 +1,299 @@
+
```

Comment:
Yah, thought that was your idea in <https://github.com/fastlane/fastlane/pull/4951#discussion_r65453186>. We could also change the `gemspec` to place the top-level `rubocop` config into the right place for the gem, I think.